### PR TITLE
	Fix issue #2662

### PR DESCRIPTION
**Pull request by starakaj**
Added a touch counter instead of flag to ensure the correct touch count

### CODE REVIEW

File: src/interaction/InteractionManager.js [hunk 1]
```diff
@@ -1327,7 +1327,11 @@ export default class InteractionManager extends EventEmitter
     {
         if (hit)
         {
-            displayObject._touchDown = true;
+            if (typeof displayObject._touchCount === 'undefined')
```
Comment (GoodBoyDigital) [inline src/interaction/InteractionManager.js]: Do we need this check? Other than that it looks good to me!

Comment (englercj) [inline src/interaction/InteractionManager.js]: Yeha looks like this `if` block should just be removed, mixing interaction target should set this to 0.

**Comment by starakaj** [comment]
Cool, I see that you are correct --- the mixin should make this check unnecessary. Made a new commit, should be good now. Thanks!

**Comment by englercj** [comment]
I think your branch is out of date, pulling in a lot of other stuff.

**Pull request closed without merging**